sources/editor/Xenko.Assets.Presentation/Templates/Core/ProjectExecutable.Linux/$ProjectName$.csproj.t4

```diff
@@ -8,6 +8,9 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace><#= Properties.Namespace #></RootNamespace>
 
+    <!-- This resolves native dep publishing -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
This will copy all assemblies to output folder (in non-publish mode).
I think `dotnet` is supposed to work without this (`dotnet` resolves assemblies from NuGet and native dependencies are loaded from within NuGet). At least it does under Windows.
IMHO Linux/macOS needs explicit publish so only publish should be fixed.
So I was wondering, with your other `CopyData` fix, isn't publish fixed already?

Reply:
Sorry, I noticed afterwise `CopyData` is only for data folder.
We should use a similar fix for native libraries. It should probably be located in `Xenko.Core\build\Xenko.Core.targets`. There is already a bunch of targets like `_XenkoSetupNativeLibrariesWindows` and `_XenkoSetupNativeLibrariesUWP` and so on. We probably want a new one `_XenkoSetupNativeLibrariesDotNetPublish`.

Reply:
Yes I missed that this impacts non-publish scenario. That's not good. Will revisit this.